Formula/dspdfviewer.rb

```diff
@@ -13,17 +12,22 @@ class Dspdfviewer < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
   depends_on "boost"
-  depends_on "poppler" => "with-qt"
+  depends_on "poppler" => "with-qt5"
 
   def install
+    args = std_cmake_args
+    args << "-DUsePrerenderedPDF=ON"
+    args << "-DRunDualScreenTests=OFF"
+    args << "-DUseQtFive=ON"
     mkdir "build" do
-      system "cmake", "..", *std_cmake_args
+      system "cmake", "..", *args
       system "make", "install"
     end
   end
 
   test do
-    assert_match /#{version}/, shell_output("#{bin}/dspdfviewer --version", 1)
+    assert_match /#{version}/, shell_output("#{bin}/dspdfviewer --version", 0)
```

Comment:
:+1: This was changed in https://github.com/dannyedel/dspdfviewer/commit/33aad7336fed14ee4802685b7b8739cf0a36e69e. Should make the test succeed now.